README.rst

```diff
@@ -59,11 +59,11 @@ You can choose from open virtualization format (OVA_, .ova), Docker_,
 or Vagrant_ appliances.
 
 --------------
-Planemo Basics
+Basics
```

Comment:
Should the section over/underlines be edited to match (six characters long)

Reply:
I don't think it matters? The older TravisCI section went over also. Is there a best practice around this?